py3status/modules/process_status.py

```diff
@@ -21,14 +27,14 @@
 import os
 import subprocess
 
-ERR_NO_PROCESS = 'no process name given'
-
 
 class Py3status:
     """
     """
     # available configuration parameters
     cache_timeout = 10
+    format = '{state}'
+    format_error = 'no process name given'
```

Comment:
I'll remove unnecessary `line 73`. I added icons. What do you want to do with `format_error`? Let me know. Thx. (and thx for helping out too).